src/core/server/base.js

```diff
@@ -230,6 +230,7 @@ function baseServer(routes, createStore, { appSagas, appInstanceName = appName }
       let htmlLang;
       let locale;
       let pageProps;
+      let runningSagas;
```

Comment:
Any reason this needs to be scoped up here? I know the scoping in this file is a bit broad and that's fine, just double-checking. The GitHub diff on this one is a bit harder to get context from.

Reply:
That's because `store` and `sagaMiddleware` are there too..

Reply:
Cool, that's what I thought, makes sense! 👍